ci/scripts/export-envvars-to-circle-ci.sh

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env bash
+set -euo pipefail
+readonly CIRCLE_API_BASE=https://circleci.com/api/v1.1/project/github/spring-cloud-incubator/spring-cloud-app-broker
+
+export_variables_to_circle() {
+	source "../.envrc"
+	#DIRECTOR_SSLCA
+	delete_envvar_from_circle DIRECTOR_SSLCA
```

Comment:
shall we not delete after the execution is done rather than on the next run?

Reply:
Actually this script is run manually using environment vars in last pass. I will add a comment explaining this. I'm not sure we need to delete at all so can remove the delete statements to avoid confusion